src/config.js

```diff
@@ -59,6 +59,35 @@ const GRANULARITY_OPTIONS = {
 
 const ALL_TOPICS = '*';
 
+const DEFAULT_IFRAMES_CONFIG = {
```

Comment:
please make this empty and then PR this as an example configuration on https://github.com/prebid/prebid.github.io/blob/master/dev-docs/modules/topicsFpdModule.md